products/ipad.md

```diff
@@ -13,6 +13,23 @@ releaseColumn: false
 # All links can be found on https://support.apple.com/HT201471.
 # All supported iPadOS versions can be found on https://en.wikipedia.org/wiki/List_of_iPad_models#iPad.
 releases:
+
+-   releaseCycle: "11"
+    releaseLabel: "iPad (11th generation)"
+    releaseDate: 2025-03-12
+    discontinued: false
+    eol: false
+    link: https://support.apple.com/122240
+    supportedIpadOsVersions: 18
+
+-   releaseCycle: "air-7"
+    releaseLabel: "iPad Air (7th generation)"
+    releaseDate: 2025-03-04
```

Comment:
```suggestion
    releaseDate: 2025-03-12
```

2025-03-04 is the announcement date, see https://www.apple.com/newsroom/2025/03/apple-introduces-ipad-air-with-powerful-m3-chip-and-new-magic-keyboard/.